src/libpsl-native/test/test-createsymlink.cpp

```diff
@@ -92,7 +92,7 @@ TEST_F(CreateSymLinkTest, SymLinkToFile)
     std::string target = FollowSymLink(fileSymLink.c_str());
     char buffer[PATH_MAX];
     std::string expected = realpath(file, buffer);
-    EXPECT_EQ(target, expected);
+    EXPECT_EQ(expected, target);
```

Comment:
Can you please explain a bit about why you made the changes to `EXPECT_EQ`?

Reply:
The pattern for `EXPECT_EQ` is `EXPECT_EQ(<expected>,<actual>)` when you call it with the parameters reverse it makes the error very confusing.